lib/beetle/message.rb

```diff
@@ -87,6 +92,8 @@ def decode #:nodoc:
       @format_version = headers[:format_version].to_i
       @flags = headers[:flags].to_i
       @expires_at = headers[:expires_at].to_i
+      @defer = headers[:defer].to_i
+      set_defer!
```

Comment:
the hack - that will be solved differently via opts and being whitelisted  (but that didnt work for whatever reason)